src/lightingCAN.cpp

```diff
@@ -1,22 +1,93 @@
 #include "lightingCAN.h"
 
+#define L_FRONT_LIGHT 0
+#define R_FRONT_LIGHT 1
+#define L_SIDE_LIGHT 2
+#define R_SIDE_LIGHT 3
+#define MID_LIGHT 4
+#define L_REAR_LIGHT 5
+#define R_REAR_LIGHT 6
+
+#define BOARD 0
+
+#if BOARD == L_FRONT_LIGHT
+#define LED_ID_1 0x300
+#define BIT_OFF 1
+#define BLINK1 1
+#define LED_ID_2 0x0//Headlight ID
```

Comment:
Could you add a `TODO: ` to these comments? That way, it's clearer that we still need to implement them.